test/powershell/engine/Job/Jobs.Tests.ps1

```diff
@@ -69,6 +69,14 @@ Describe 'Basic Job Tests' -Tags 'CI' {
             $ProgressMsg[0].StatusDescription | Should -BeExactly 2
         }
 
+        It 'Can use the user specified working directory parameter' {
```

Comment:
Also, please add a test for an invalid working directory path.

Reply:
fixed in 95ab95d2d5c043a1fbc3ba78f1ff94c9e45d94af